tdvt/test/tool_test/test_generation/full_map_to_calcs.csv

```diff
@@ -0,0 +1,4 @@
+k,n0,n1,n2,n3,n4,s0,s1,s2,s3,i0,i1,i2,i3,b0,b1,b2,b3,d0,d1,d2,d3,t0,t1,dt0,dt1,z
```

Comment:
For tables with lots of columns its going to be easy to get misaligned on these rows. Should we consider json or some other way to group multiple properties for a single column.

Reply:
That is a much better user experience, I'll refactor to use json.

Reply:
Updated below.